.env.example

```diff
@@ -1,20 +0,0 @@
-# This file serves as a template for configuration through environment
-# variables. It should always be sufficient for use in development and
-# testing. It should also be always complete in a sense that all
-# variables that can be set are included. It is used to extract a list
-# of permitted variables for production during deployment, commented out
-# or not.
-export SECRET_KEY_BASE=foo
-export SECRET_TOKEN=bar
-export RAILS_ENV=development
-export POSTGRES_URL="postgres://localhost/snpr_$RAILS_ENV"
-export REDIS_URL=redis://localhost
-export FITBIT_CONSUMER_KEY=foo
-export FITBIT_CONSUMER_SECRET=bar
-export PLOS_API_KEY=foo
-export RECAPTCHA_PRIVATE_KEY=6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
-export RECAPTCHA_PUBLIC_KEY=6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI
-export SIDEKIQ_PASSWORD=foo
-export MENDELEY_CONSUMER_KEY=foo
-export ERRBIT_API_KEY=foo
-export ERRBIT_HOST=localhost
```

Comment:
`.env` is in the `.gitignore` file. The idea is that the example file works for testing and to run the app, but you still have the option to change your `.env` without having to take care of not accidentally commit any API keys you don't want to publish.